core/src/main/kotlin/api/AdminRoute.kt

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2022 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.eclipse.apoapsis.ortserver.core.api
+
+import io.github.smiley4.ktorswaggerui.dsl.routing.get
+
+import io.ktor.http.HttpStatusCode
+import io.ktor.server.application.Application
+import io.ktor.server.application.call
+import io.ktor.server.response.respond
+import io.ktor.server.routing.Route
+import io.ktor.server.routing.route
+
+import kotlinx.coroutines.launch
+
+import org.eclipse.apoapsis.ortserver.core.apiDocs.runPermissionsSync
+import org.eclipse.apoapsis.ortserver.core.authorization.requirePermission
+import org.eclipse.apoapsis.ortserver.model.authorization.Superuser
+import org.eclipse.apoapsis.ortserver.services.AuthorizationService
+import org.eclipse.apoapsis.ortserver.utils.logging.withMdcContext
+
+import org.koin.ktor.ext.inject
+
+import org.slf4j.LoggerFactory
+
+private val logger = LoggerFactory.getLogger(Application::class.java)
+
+fun Route.admin() = route("admin") {
+    route("sync-roles") {
+        val authorizationService by inject<AuthorizationService>()
+
+        get(runPermissionsSync) {
+            requirePermission(Superuser.ROLE_NAME)
+
+            withMdcContext("component" to "core") {
```

Comment:
This is not required, it is automatically done for all calls by the `CallLogging` plugin configured in `Monitoring.kt`.
Instead, use `withContext(Dispatchers.IO)`.

Reply:
Not yet changed.